composer.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "salcode/bootstrap-genesis",
+    "description": "WordPress Genesis Child Theme setup to use Bootstrap, Sass, and Grunt",
+    "keywords": ["wordpress", "theme"],
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Sal Ferrarello",
+            "email": "sal@ironcodestudio.com",
+            "homepage": "http://salferrarello.com/"
+        }
+    ],
+    "autoload": {
```

Comment:
Actually I'm not a WordPress master, so I don't know how you are about to load the classes, I was thinking about something like this `require_once __DIR__.'/vendor/autoload.php';` in a single place.
What do you think about https://github.com/composer/installers ? do you need to install the theme in a custom folder?